devicon.json

```diff
@@ -2238,6 +2238,26 @@
             }
         ]
     },
+    {
+        "name": "detaspace",
+        "altnames": [
+            "Deta Space"
+        ],
+        "tags": [
+            "cloud",
+            "hosting",
+            "server"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": []
```

Comment:
Could you please provide a plain version, so that we can have something for the font as well?
Make sure it has a single path and a single color. Maybe it could just be the same as the original version but with some space inbetween each ring?